src/account/account.cairo

```diff
@@ -245,4 +246,89 @@ mod AccountComponent {
             is_valid_stark_signature(hash, public_key, signature)
         }
     }
+
+    #[embeddable_as(AccountABIImpl)]
+    impl AccountABI<
```

Comment:
why `ABI`? some alternatives:

`AccountMixin`
`BaseAccount`
`BasicAccount`
`FullAccount` (might sound contradictory with the previous one, but both convey they are enough)
`Account`, to favor in the eyes of users, then we need to rename every other impl

whatever we decide needs to be applied to all other mixins

Reply:
I originally had it as `<component>Mixin`, but I noticed that the mixin interface matched the `ABI` trait type. Then, I thought it'd be more organized if the ABI name matched with the mixin impls. I changed it back to `Mixin`. Let me know what you think